cloud/amazon/cloudformation.py

```diff
@@ -14,12 +14,22 @@
 # You should have received a copy of the GNU General Public License
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 
+# upcoming features:
```

Comment:
I'd recommend not committing a TODO list into the file, though all these would be great features to have :smile: